lumicks/pylake/force_calibration/calibration_results.py

```diff
@@ -75,10 +75,21 @@ def __repr__(self):
         return f"{self.__class__.__name__}({properties})"
 
     @property
-    def _fit_range(self):
+    def _fit_range(self) -> tuple[float, float]:
         """Fitted range"""
         return self.ps_data._fit_range
 
+    @property
+    def _initial_fit_range(self) -> tuple[float, float] | None:
+        """Initial fit range"""
+        if "Initial fit range (min.)" not in self.params:
```

Comment:
Same as above.

Reply:
Updated to require both and raise if only one exists.